frontend/src/entities/emr/detail/emr-clusters-detail.tsx

```diff
@@ -63,6 +64,11 @@ function EMRDetail () {
         }
     }, [dispatch, basePath, projectName, clusterId, clusterName]);
 
+    // Refresh data in the background to keep state fresh
+    const isBackgroundRefreshing = useBackgroundRefresh(() => {
```

Comment:
Just my own preference, but I don't think we should be doing background refreshes on entities which are not prone to frequent updates. 

I think auto-refreshing on jobs (training, translate, transform) is great and that you've implemented this in a very nice way, but resources like EMR and Notebooks change state very infrequently and my concern would be that if someone left the EMR/Notebook detail view open and then left their computer, they'd run up a bill from these repeated describe calls. 

Out of scope for this PR/task would be adding a little refresh button to the Notebook and EMR pages, whereas for jobs it's nice that we check for their status on an interval and then have a definite stop condition when the job has reached a terminal state.

Reply:
This is similar to the AWS console. While technically in the console you aren't charged the cost in MLSpace would be negligible - $0.2583 per million background refreshes based on ddb read and lambda invocations (prices based on us-east-2). If customers are concerned they can increase the frequency of refreshes. This feature is in direct response to a feature request from multiple customers.